go/brute_force/brute_force.go

```diff
@@ -0,0 +1,92 @@
+package brute_force
+
+// #include <cuda_runtime_api.h>
```

Comment:
Can these be removed?

Reply:
The Go compiler uses these comments to link the necessary cuvs source files: https://go.dev/wiki/cgo